chromium/.eslintrc.json

```diff
@@ -1,16 +1,16 @@
 {
   "env": {
     "browser": true,
-    "commonjs": true,
     "es6": true,
-    "node": true,
     "webextensions": true
   },
   "parserOptions": {
     "ecmaVersion": 2017,
     "sourceType": "script"
   },
   "rules": {
-    "indent": [2, 2]
+    "indent": ["error", 2],
```

Comment:
what does this change?

Reply:
`"indent": [2, 2]` is not a valid value.